osu.Game/Screens/Play/PlayerSettings/InputSettings.cs

```diff
@@ -19,12 +20,13 @@ public InputSettings()
             {
                 mouseButtonsCheckbox = new PlayerCheckbox
```

Comment:
I'd rename the variable to `disableGameplayInputCheckbox`.

Reply:
👍 
Missed that one in the refactor. But I think I'll just move this to BDL and scrap the variable entirely.